server/src/app.ts

```diff
@@ -2,12 +2,23 @@
 import dotenv from "dotenv"
 dotenv.config()
 
-import express, { Express, Request, Response } from "express"
+import express, { Application, Request, Response, NextFunction } from "express";
+
+import { router as userRoutes } from "./routes/user.routes";
+
+
 import cors from "cors"
 
-const app: Express = express()
+const app: Application = express();
 const port: number | string | undefined = process.env.PORT || 4001
```

Comment:
Seems like we are not using that port, so we can delete it?

Reply:
I will delete it, thanks!